types/validator_set.go

```diff
@@ -361,7 +349,11 @@ func processChanges(origChanges []*Validator) (updates, removals []*Validator, e
 			return nil, nil, err
 		}
 		if valUpdate.VotingPower < 0 {
-			err = fmt.Errorf("voting power can't be negative %v", valUpdate)
+			err = fmt.Errorf("voting power can't be negative: %v", valUpdate)
+			return nil, nil, err
+		}
+		if valUpdate.VotingPower > MaxTotalVotingPower {
+			err = fmt.Errorf("voting power can't be higher than %v: %v ", MaxTotalVotingPower, valUpdate)
```

Comment:
do you think we also should explain WHY in the error msg? or it will make the msg too long?

Reply:
I added something, let me know.